Dockerfile

```diff
@@ -304,13 +335,12 @@ SHELL ["/bin/bash", "-o", "pipefail","-e", "-x", "-c"]
 
 WORKDIR /platform
 
-COPY --from=build-planner /platform/recipe.json recipe.json
+COPY --from=build-planner /platform/recipe.json /platform/.cargo /platform/
 
 # Build dependencies - this is the caching Docker layer!
 RUN --mount=type=cache,sharing=shared,id=cargo_registry_index,target=${CARGO_HOME}/registry/index \
     --mount=type=cache,sharing=shared,id=cargo_registry_cache,target=${CARGO_HOME}/registry/cache \
     --mount=type=cache,sharing=shared,id=cargo_git,target=${CARGO_HOME}/git/db \
-    #--mount=type=cache,sharing=shared,id=target_${TARGETARCH},target=/platform/target \
```

Comment:
you're not cache-mounting /target anymore. Does it mean it will get into layer cache, and will be uploaded/downloaded to s3?

Reply:
Yes, we want to cache the target in layers to reuse it in further instructions.